tests/test_06.R

```diff
@@ -0,0 +1,26 @@
+# copyright (c) - HU-Berlin / UTHSC / JICS by Danny Arends
+
+# Compare MPCC versus cor() function, mA = 15 x 5, mB = 15 x 10, no missing data
+library(MPCC)
+MPCCinfo()
+
+set.seed(1)
+mAB <- genAB(p = 200, n = 2000, m = 100)
+
+s.ref <- proc.time()[3]
+ref <- cor(mAB[["A"]], mAB[["B"]])
+(proc.time()[3] - s.ref)
+
+s.mpcc <- proc.time()[3]
+mpcc <- PCC(mAB[["A"]], mAB[["B"]])
+(proc.time()[3] - s.mpcc)
+
+s.mpcc <- proc.time()[3]
+mpcc <- PCC.naive(mAB[["A"]], mAB[["B"]])
+(proc.time()[3] - s.mpcc)
+
+
+if (sum(round(mpcc - ref, 12)) != 0) {
```

Comment:
The mpcc variable is overwritten, you're not testing PCC, but PCC.naive
Store the results in different variables and use 2 if statement to check both PCC and PCC.naive

Reply:
Yeah, will add another if statement.